package.json

```
@@ -16,7 +16,7 @@
     "access": "public"
   },
   "engines": {
-    "node": "14 || 16 || 17 || 18 || 19"
+    "node": "14 || 16 || 17 || 18 || 19 || 20"
```

What about this? @achrinza :wink:


```suggestion
    "node": ">=14"
```